.pre-commit-config.yaml

```diff
@@ -0,0 +1,33 @@
+repos:
+- repo: https://github.com/psf/black
+  rev: 22.1.0
+  hooks:
+  - id: black
+- repo: https://github.com/asottile/reorder_python_imports
+  rev: v2.7.1
+  hooks:
+  - id: reorder-python-imports
+    args: ["--py37-plus"]
+- repo: https://gitlab.com/PyCQA/flake8
+  rev: 3.9.2
+  hooks:
+  - id: flake8
+- repo: https://github.com/codespell-project/codespell
+  rev: v2.1.0
+  hooks:
+  - id: codespell
+# - repo: https://github.com/pycqa/pydocstyle
```

Comment:
Currently disabled because the docstrings in this repo don't conform to this style. Maybe we can enable and fix this in a separate PR?

Reply:
I agree, otherwise the PR can be confusing if it implies too many modifications.